LocalSettings.php

```diff
@@ -2659,15 +2659,19 @@
 				'suppressionlog' => true,
 				'suppressrevision' => true,
 			],
-			'user' => [
-				'mwoauthmanagemygrants' => true,
-				'user' => true,
-			],
 			'steward' => [
+				'centralauth-createlocal' => true,
 				'centralauth-usermerge' => true,
 				'usermerge' => true,
 				'userrights' => true,
 			],
+			'sysop' => [
+				'centralauth-createlocal' => false,
```

Comment:
=> false == do nothing at all

Reply:
Oh. Thanks! I'll have to use WgRevokePermissions then? Or wgGroupPermissions directly instead of this?

Reply:
Neither? What are you trying to achieve?

Reply:
Revoke it from sysop since the extension adds it to sysop by default, blacklisting it won't matter since sysop would still have it.